source/agora/consensus/EnrollmentManager.d

```diff
@@ -104,7 +102,7 @@ public class EnrollmentManager
 
     /// Validator set managing validators' information such as Enrollment object
     /// enrolled height, and preimages.
-    private ValidatorSet validator_set;
+    public ValidatorSet validator_set;  // FIXME: Made public to ease transition to raise this to ledger
```

Comment:
No change in this file seems related to the PR ?

Reply:
Well to enable removing the count without a height parameter these changes are all needed.

Reply:
I will update the commit message to describe why the stats had to move also.

Reply:
Or I could extract the move of stats into a separate commit if you prefer?

Reply:
> Or I could extract the move of stats into a separate commit if you prefer?

Yep

Reply:
Updated